spec/acceptance/init_spec.rb

```diff
@@ -1,74 +1,75 @@
 # run a test task
 require 'spec_helper_acceptance'
+require 'pry'
 
 describe 'package task' do
-  include Beaker::TaskHelper::Inventory
-  include BoltSpec::Run
-
   operating_system_fact = os[:family]
   redhat_six = os[:family] == 'redhat' && os[:release].to_i == 6
 
-  def bolt_config
-    { 'modulepath' => RSpec.configuration.module_path }
+  before(:all) do
+    inventory_hash = inventory_hash_from_inventory_file
+    inventory_hash = add_feature_to_group(inventory_hash, 'puppet-agent', 'ssh_nodes')
+    inventory_hash = add_feature_to_group(inventory_hash, 'puppet-agent', 'winrm_nodes')
+    inventory_hash = add_feature_to_group(inventory_hash, 'puppet-agent', 'docker_nodes')
+    write_to_inventory_file(inventory_hash, 'inventory.yaml')
   end
 
-  let(:bolt_inventory) { hosts_to_inventory.merge('features' => ['puppet-agent']) }
-
   describe 'install' do
     before(:all) do
-      apply_manifest_on(default, 'package { "pry": ensure => absent, provider => "puppet_gem", }')
+      apply_manifest('package { "pry": ensure => absent, provider => "puppet_gem", }')
     end
 
     it 'installs pry', unless: (operating_system_fact == 'windows') do
-      result = run_task('package', 'default', 'action' => 'install', 'name' => 'pry', 'provider' => 'puppet_gem')
```

Comment:
Would it make sense to just use the `BoltSpec::Run` methods directory for all the Rspec tests with the inventory and config setup by litmus?

Reply:
@donoghuc Let me go through the BoltSpec:: Run methods. Sounds good.

Reply:
@donoghuc Looks like we use bolt_spec/run' https://github.com/puppetlabs/puppet_litmus/blob/master/lib/puppet_litmus.rb#L6

Reply:
Yeah, so my suggestion would be to use for example the BoltSpec::Run.run_task method directly in the rspec tests instead of the `run_bolt_task` method from litmus https://github.com/puppetlabs/puppet_litmus/blob/7c7bcb5e4b5d6202d1e7f10a678e810679a02845/lib/puppet_litmus/serverspec.rb#L142 

Specifically, in general when your tests need the "bolt native" structure that is returned by for example https://github.com/puppetlabs/bolt/blob/45056d869f7377e196e6b0aeb177d3515ed3d63f/lib/bolt_spec/run.rb#L27 you should just use the BoltSpec::Run methods directly. When you need to use the litmus method (where you dont necessarily care about the task details, and only that it succeded/failed) then use the litmus `run_bolt_task` method.

Reply:
@donoghuc we have updated run_bolt_task to handle result hash too.So we can test all the values returned from hash.
https://github.com/puppetlabs/puppet_litmus/blob/master/lib/puppet_litmus/serverspec.rb#L159